src/backfill/index.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable prefer-const */
```

Comment:
these methods and `poolMappings.ts` are only used by optimism to seed pre-regenesis data